connector/MongoDBAtlasODBC.pq

```diff
@@ -73,3 +81,64 @@ MongoDBAtlasODBC.Icons = [
     Icon16 = { Extension.Contents("MongoDBAtlasODBC16.png"), Extension.Contents("MongoDBAtlasODBC20.png"), Extension.Contents("MongoDBAtlasODBC24.png"), Extension.Contents("MongoDBAtlasODBC32.png") },
     Icon32 = { Extension.Contents("MongoDBAtlasODBC32.png"), Extension.Contents("MongoDBAtlasODBC40.png"), Extension.Contents("MongoDBAtlasODBC48.png"), Extension.Contents("MongoDBAtlasODBC64.png") }
 ];
+
+BuildOdbcConfig = () as record =>
+    let
+        Config = [
+            SqlCapabilities = [
+                FractionalSecondsScale = 3,
+                LimitClauseKind = LimitClauseKind.LimitOffset,
+                SupportsDerivedTable = true,
+                SupportsNumericLiterals = true,
+                SupportsStringLiterals = true,
+                SupportsOdbcDateLiterals = true,
+                SupportsOdbcTimeLiterals = true,
+                SupportsOdbcTimestampLiterals = true

```

Comment:
Will we need to specify somewhere _how_ these literals are written? I think I missed out on asking about this in the design doc. Will PowerBI by default write the literals using CASTs like we require, or will they try to use some other syntax? Can we specify this and is that work ticketed already?

Reply:
That's a good point.  There is a way to modify the AstVisitor ([example](https://github.com/microsoft/DataConnectors/blob/395c34ccbe1ca21675a8244715820a26da8eb622/samples/ODBC/SnowflakeODBC/SnowflakeODBC.pq#L32)) and when encountering TIMESTAMP for example to add a cast with the specified syntax.  It looks like that's what we will need, I created a ticket to implement the AstVisitor: SQL-1144.